telemetry/telemetryformat.md

```diff
@@ -1,19 +1,20 @@
 # Telemetry Format
 
-Telemetry definitions are made up of json files. Multiple files can exist and are merged together at build time. After they are merged, one file in the target language is generated that can call the platform specific telemetry client. Each platform's generator has platform specific ways to generate a final telemetry file, see each platform's readme for more details.
+Telemetry definitions are made up of json files that live in `definitions`. Files are merged together at build time. They output one file in the target language that can call the platform specific telemetry client. Each platform's generator has platform specific tweaks in 
```

Comment:
weird line break at the end of this line + start of next line

Reply:
oh weird, fixed